app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -263,38 +268,39 @@ public static void SendLibrereading(final String tagId, byte[] data1, final long
         }
         // Create the object to send
         libreBlock = LibreBlock.create(tagId, CaptureDateTime, data1, 0, patchUid, patchInfo);
-        if(libreBlock == null) {
+        if (libreBlock == null) {
             Log.e(TAG, "Error could not create libreBlock for libre-allhouse");
             return;
         }
         final String json = libreBlock.toExtendedJson();
-        
+
         GcmActivity.pushLibreBlock(json);
-    
+
     }
 
-    public static boolean HandleGoodReading(final String tagId, byte[] data1, final long CaptureDateTime, final boolean allowUpload, byte []patchUid,  byte []patchInfo ) {
-        return HandleGoodReading(tagId, data1, CaptureDateTime, allowUpload, patchUid,  patchInfo, false ) ;
+    public static boolean HandleGoodReading(final String tagId, byte[] data1, final long CaptureDateTime, final boolean allowUpload, byte[] patchUid, byte[] patchInfo) {
+        return HandleGoodReading(tagId, data1, CaptureDateTime, allowUpload, patchUid, patchInfo, false, null, null);
     }
-    
-    
+
+
     // returns true if checksum passed.
-    public static boolean HandleGoodReading(final String tagId, byte[] data1, final long CaptureDateTime, final boolean allowUpload, byte []patchUid,  byte []patchInfo, boolean decripted_data ) {
+    public static boolean HandleGoodReading(final String tagId, byte[] data1, final long CaptureDateTime, final boolean allowUpload, byte[] patchUid, byte[] patchInfo,
+                                            boolean decripted_data, int[] trend_bg_vals, int[] history_bg_vals) {
         Log.e(TAG, "HandleGoodReading called dat1 len = " + data1.length);
-        if(data1.length > Constants.LIBRE_1_2_FRAM_SIZE) {
+        if (data1.length > Constants.LIBRE_1_2_FRAM_SIZE) {
             // It seems that some times we read a buffer that is bigger than 0x158, but we should only use the first 0x158 bytes.
             data1 = java.util.Arrays.copyOfRange(data1, 0, Constants.LIBRE_1_2_FRAM_SIZE);
         }
 
-        if(LibreOOPAlgorithm.isDecodeableData(patchInfo) && decripted_data == false
+        if (LibreOOPAlgorithm.isDecodeableData(patchInfo) && decripted_data == false
                 && !Pref.getBooleanDefaultFalse("external_blukon_algorithm")) {
             // Send to OOP2 for decryption.
             LibreOOPAlgorithm.logIfOOP2NotAlive();
             LibreOOPAlgorithm.sendData(data1, CaptureDateTime, patchUid, patchInfo, tagId);
             return true;
         }
 
-        SendLibrereading(tagId, data1, CaptureDateTime, patchUid, patchInfo);
+        SendLibrereadingToFollowers(tagId, data1, CaptureDateTime, patchUid, patchInfo);
```

Comment:
If you touch this function name, please change it to full camel-case except first character like `sendLibreReadingsToFollowers'. See [Java Naming Conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) for details.

Reply:
Done.